### PR TITLE
solution to catalog

### PR DESCRIPTION
**Pull request by mykola-koval**
- [DEMO LINK](https://mykola-koval.github.io/layout_catalog/)
- [TEST REPORT LINK](https://mykola-koval.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,355 @@
...
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="moyo"
```
Comment (ArtemTopolskyi) [inline src/index.html:28]:
```suggestion
          alt="Moyo logotype"
```

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,355 @@
...
+        </ul>
+      </nav>
+    </header>
+
+    <div class="main page__main">
+
+      <div class="card main__card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt=""
```
Comment (ArtemTopolskyi) [inline src/index.html:71]: do not left alt attribute empty

File: src/index.html [hunk 3]
```diff
@@ -4,9 +4,355 @@
...
+          <span class="card__pricetag-price">69 999 грн</span>
+        </div>
+
+        <a
+          class="card__buy-button"
+          href="#buy"
+          data-qa="card-hover"
+        >Купить
+        </a>
+      </div>
```
Comment (ArtemTopolskyi) [inline src/index.html:103]:
```suggestion
        >
          Купить
        </a>
      </div>
```

File: src/styles/blocks/header.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+
+  &__logo {
+    height: 40px;
+  }
+}
+
+.nav {
```
Comment (ArtemTopolskyi) [inline src/styles/blocks/header.scss:15]: these styles must be in separate 'nav.scss' file

**Review by ArtemTopolskyi — CHANGES_REQUESTED** [review]

**Comment by mykola-koval** [comment]
npm run deploy seems not working
any ideas?

**Review by ArtemTopolskyi — APPROVED** [review]

**Pull request closed without merging**